lms/static/coffee/src/customwmd.coffee

```diff
@@ -31,7 +31,9 @@ $ ->
       block = @blocks.slice(start, last + 1).join("").replace(/&/g, "&amp;")
                                                     .replace(/</g, "&lt;")
                                                     .replace(/>/g, "&gt;")
-      if HUB.Browser.isMSIE
+      # Using MathJax.Hub directly instead of creating HUB variable, Sometimes it is called before
+      # being initialized (TNL-1902).
```

Comment:
This comment is actually not needed in the code; mentioning it in the PR would have been sufficient.